packages/injected/README.md

```
@@ -1 +1,61 @@
 # cross-inpage-provider
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)